ADAL/src/cache/ios/ADKeychainTokenCache.m

```diff
@@ -411,7 +413,7 @@ - (BOOL)removeAllForUserId:(NSString * __nonnull)userId
                      error:(ADAuthenticationError * __nullable __autoreleasing * __nullable)error
 {
     AD_LOG_WARN(nil, @"Removing all items for user + client <%@>", clientId);
```

Comment:
clientID is OII, we cannot log it in a regular log

Reply:
fixed